Kernel/Graphics/VirtIOGPU/Console.h

```diff
@@ -42,8 +42,8 @@ class Console final : public GenericFramebufferConsole {
     void enqueue_refresh_timer();
     virtual u8* framebuffer_data() override;
 
-    Console(RefPtr<FrameBufferDevice> const&);
-    RefPtr<FrameBufferDevice> m_framebuffer_device;
+    Console(RefPtr<Graphics::VirtIOGPU::FramebufferDevice> const&);
+    RefPtr<FramebufferDevice> m_framebuffer_device;
```

Comment:
Is there a reason why we aren't changing this to `RefPtr<Graphics::VirtIOGPU::FramebufferDevice> m_framebuffer_device;` ?